src/App.tsx

```diff
@@ -1,24 +1,251 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { changeTodo, createTodo, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './TodoList';
+import { Status } from './types/Status';
+import { TodosFilter } from './TodosFilter';
+import { CompletedChangeTodo } from './types/CompletedChangeTodo';
 
-const USER_ID = 0;
+const USER_ID = 11623;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState(Status.all);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [value, setValue] = useState('');
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [changedTodos, setChangedTodos] = useState<number[]>([]);
+  const [editFlag, setEditFlag] = useState<boolean>(false);
+
+  const isAllCompleted = todos.every((todo: Todo) => todo.completed);
+
+  const inputRef = useRef<HTMLInputElement | null>(null);
+
+  useEffect(() => {
+    if (inputRef.current) {
+      inputRef.current.focus();
+    }
+  }, [todos.length]);
+
+  useEffect(() => {
+    if (errorMessage && !editFlag && inputRef.current) {
+      inputRef.current.focus();
+    }
+  }, [errorMessage]);
```

Comment:
fix this linter issue
![Screenshot from 2023-10-10 15-08-32](https://github.com/mate-academy/react_todo-app-with-api/assets/33029864/e77cc49c-855d-46d4-bbe5-7078a3fe6126)